web-api/src/models.rs

```diff
@@ -125,13 +125,13 @@ fn is_valid_property_id(value: &str) -> bool {
 fn is_valid_string(value: &str, len: usize) -> bool {
     static RE: Lazy<Regex> = Lazy::new(|| Regex::new(r"^[^\x00]+$").unwrap());
 
-    (1..=len).contains(&value.len()) && RE.is_match(value)
+    RE.is_match(value)
```

Comment:
:art: this is the quick fix you mentioned in the daily, right? this is not supposed to be merged later on.